Database/Corrections/cataQuestFixes.lua

```diff
@@ -7352,7 +7352,7 @@ function CataQuestFixes.Load()
             [questKeys.preQuestGroup] = {27625,27626},
         },
         [26642] = { -- Preserving the Barrens
-            [questKeys.exclusiveTo] = {},
+            [questKeys.exclusiveTo] = {28494},
```

Comment:
@Cabro tested these in great detail and is sure you can complete both (even though you might not be able to have both in your quest log).

```suggestion
            [questKeys.exclusiveTo] = {},
```